packages/common/src/intl/IntlProvider.tsx

```diff
@@ -6,9 +6,15 @@ import HttpApi from 'i18next-http-backend';
 import { I18nextProvider, initReactI18next } from 'react-i18next';
 import LanguageDetector from 'i18next-browser-languagedetector';
 
-const defaultNS = 'app';
+const defaultNS = 'common';
 export const IntlProviderNext: React.FC = ({ children }) => {
   React.useEffect(() => {
+    const minuteInMilliseconds = 60 * 1000;
+    const expirationTime =
```

Comment:
the caching default is one week and this is annoying when developing, so I've changed to 5 minutes. possibly still too much? Have left the production value at a week 🤷